modules/augment/include/opencv2/augment/rotate.hpp

```diff
@@ -0,0 +1,55 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+
+#ifndef OPENCV_AUGMENT_ROTATE_HPP
+#define OPENCV_AUGMENT_ROTATE_HPP
+#include <opencv2/augment/Transform.hpp>
+
+namespace cv {
+namespace augment {
+
+    //! @addtogroup augment
+        //! @{
+
+    class CV_EXPORTS_W Rotate : public Transform
+    {
+    public:
+        /* @brief Constructor
+           @param _angleRange the range of angle values to rotate the image
+        */
+        CV_WRAP Rotate(Range _angleRange);
```

Comment:
Please do not use extra characters if it's not necessary. Just keep things simpler: `Range angleRange`.

Reply:
done